src/Libm.jl

```diff
@@ -1,5 +1,80 @@
 module Libm
 
-# package code goes here
+export xexp
 
-end # module
+const PI4_A = 0.78539816290140151978
+const PI4_B = 4.9604678871439933374e-10
+const PI4_C = 1.1258708853173288931e-18
+const PI4_D = 1.7607799325916000908e-27
+
+const M_4_PI = 1.273239544735162542821171882678754627704620361328125
+
+const L2U = .69314718055966295651160180568695068359375
+const L2L = .28235290563031577122588448175013436025525412068e-12
+const R_LN2 = 1.442695040888963407359924681001892137426645954152985934135449406931
+
+@inline double_to_raw_long_bits(d::Float64) = reinterpret(Int64, d)
```

Comment:
I suppose `reinterpret` is always translated directly to LLVM IR instruction, so no need to care about inlining. See e.g.
```
xx(x::Float64) = reinterpret(Float64, reinterpret(Int64, x)^2)

julia> @code_llvm xx(5.0)

define double @julia_xx_66265(double) #0 !dbg !6 {
top:
  %1 = bitcast double %0 to i64
  %2 = call i64 @jlsys_power_by_squaring_41429(i64 %1, i64 2)
  %3 = bitcast i64 %2 to double
  ret double %3
}
```